### PR TITLE
sriov pkg: WithLogLevel added to the sriovnetwork

### PR DESCRIPTION
**Pull request by evgenLevin**

### CODE REVIEW

**Review by kononovn — DISMISSED** [review]
/lgtm

File: pkg/sriov/network.go [hunk 1]
```diff
@@ -222,6 +222,30 @@ func (builder *NetworkBuilder) WithVlanQoS(qoSClass uint16) *NetworkBuilder {
...
+	if !slices.Contains(allowedLogLevels, logLevel) {
+		builder.errorMsg = "invalid logLevel value, allowed logLevel values are:" +
+			" panic, error, warning, info, debug or empty"
+
+		return builder
+	}
+
+	if builder.errorMsg != "" {
+		return builder
+	}
```
Comment (trewest) [inline pkg/sriov/network.go:242]: This is redundant

Comment (evgenLevin) [reply pkg/sriov/network.go]: removed

**Review by trewest — APPROVED** [review]
/lgtm

**Pull request merged**